.travis.yml

```diff
@@ -95,7 +95,12 @@ jobs:
         - git diff --stat --exit-code .travis.yml
 
     - stage: Deploy
-      script: ./generate-stackbrew-pr.sh
+      script:
+        - |
+          if [ "false" != "$TRAVIS_PULL_REQUEST" ]; then
```

Comment:
Prefer not to use double negative statement here which would be easier to read/understand for me.

Reply:
This was copied from the skip logic. Maybe we have a separate PR for addressing these?

Reply:
Okay, sure. Maybe it's a bad example from me, or maybe there is a concern that I didn't recall yet. So even if we'd like to change this, well change it separately later.